src/components/SeparatorVirtualGrid.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+	<h2 class="grid-title" :style="{height: item.height + 'px'}">
+		{{ item.injected.month }}
+		<span>{{ item.injected.year }}</span>
+	</h2>
+</template>
+
+<script>
+export default {
+	name: 'SeparatorVirtualGrid',
+
+	inheritAttrs: false,
+
+	props: {
+		item: {
+			type: Object,
+			required: true,
+			injected: {
+				type: Object,
+				required: true,
+				month: {
+					type: String,
+					required: true,
+				},
+				year: {
+					type: String,
+					required: true,
+				},
+			},
+			height: {
+				type: Number,
+				required: true,
+			},
+		},
+	},
+}
+</script>
+
+<style lang="scss" scoped>
+.grid-title {
```

Comment:
Grid title should be using the grid config sizing (padding, margins...)

Reply:
Yes understood